Bitmark/Services/BitmarkSDKService.swift

```diff
@@ -32,14 +32,14 @@ class BitmarkSDKServiceLogger: SDKLogger {
     Global.log.logln(message,
                      level: sdkToAppLogLevel(level),
                      functionName: "",
-                     fileName: "",
+                     fileName: "BitmarkSDK",
                      lineNumber: 0,
-                     userInfo: ["Source": "BitmarkSDK"])
+                     userInfo: [:])
 
     if level == .error {
```

Comment:
in this case, I think we can remove if condition, and only execute 
```Global.log.info(message)```

Because some cases of BitmarkSDK's error, we don't want to notify Sentry( example: asset not found when user register new asset).
other cases our project should catch and report error if needed.

Reply:
Ok